### PR TITLE
Feature/nexteuropa 5213 implement a module to manage

### PR DESCRIPTION
**Pull request by champcy**

### CODE REVIEW

File: profiles/common/modules/features/nexteuropa_geofield/nexteuropa_geofield.module [hunk 1]
```diff
@@ -0,0 +1,327 @@
...
+        );
+      break;
+
+    case 'geofield_geojson_map_format':
+        $value = isset($items[0]['geofield_geojson']) ? $items[0]['geofield_geojson'] : '';
+      $element[0] = array(
+        '#type' => 'markup',
+        '#markup' => '<div id="geofield_geojson_map"></div><div id="geofield_geojson_map_wrapper"></div>',
+      );
+      $element[0]['#attached']['js'][0] = array(
```
Comment (senseca) [inline profiles/common/modules/features/nexteuropa_geofield/nexteuropa_geofield.module]: Page not found : 
[...]profiles/multisite_drupal_standard/libraries/leaflet/leaflet-src.js?nugaoq

So the map is not available on the content form...

Comment (champcy) [inline profiles/common/modules/features/nexteuropa_geofield/nexteuropa_geofield.module]: The branch has been cleaned, you can check again.

File: profiles/common/modules/features/nexteuropa_geofield/README.txt [hunk 2]
```diff
@@ -0,0 +1,28 @@
+INTRODUCTION
+------------
+This module provides 
+- a new field type (geofield geoJSON) to store geoJSON data
+- a widget with an interactive map and tools to add objects on the map   
+- the geojson data (in json format) can be get through an URL : node/X/geojson
+
+REQUIREMENTS
+------------
+The libraries leaflet and leaflet.draw are required.
```
Comment (senseca) [inline profiles/common/modules/features/nexteuropa_geofield/README.txt]: Maybe we can add a hook_requirement pour check if the library is available?

File: profiles/common/modules/features/nexteuropa_geofield/nexteuropa_geofield.info [hunk 3]
```diff
@@ -0,0 +1,8 @@
+name = nexteuropa_geofield
```
Comment (senseca) [inline profiles/common/modules/features/nexteuropa_geofield/nexteuropa_geofield.info]: Can we rename 'Nexteuropa geofield' (to be compliant with other features)?

File: profiles/common/modules/features/nexteuropa_geofield/nexteuropa_geofield.module [hunk 4]
```diff
@@ -0,0 +1,327 @@
...
+function nexteuropa_geofield_field_widget_info() {
+
+  return array(
+    'geofield_widget_map' => array(
+      'label' => t('Interactive map'),
+      'field types' => array('geofield_geojson'),
+      /*'behaviors' => array(
+        'multiple values' => FIELD_BEHAVIOR_CUSTOM,
+        'default value' => FIELD_BEHAVIOR_NONE,
+      ),*/
```
Comment (senseca) [inline profiles/common/modules/features/nexteuropa_geofield/nexteuropa_geofield.module]: Please, remove useless comments.

File: profiles/common/modules/features/nexteuropa_geofield/nexteuropa_geofield.module [hunk 5]
```diff
@@ -0,0 +1,327 @@
...
+      // '#other' => t('Free value'),.
+      '#default_value' => isset($value['fs_objects']['fs_prepopulate']['name_populate']) ? $value['fs_objects']['fs_prepopulate']['name_populate'] : 'title_field',
+    );
+    $element['fs_objects']['fs_prepopulate']['description_populate'] = array(
+      // '#type' => 'select_or_other',.
+      '#type' => 'select',
+      '#title' => t('Popup description'),
+      '#options' => $options,
+      '#description' => t('Choose a field to populate the popup description'),
+      // '#other' => t('Free value'),.
```
Comment (senseca) [inline profiles/common/modules/features/nexteuropa_geofield/nexteuropa_geofield.module]: l. 180, l189 : Please, remove useless comments.

File: resources/multisite_drupal_standard.make [hunk 6]
```diff
@@ -275,11 +275,18 @@ projects[l10n_update][subdir] = "contrib"
 projects[l10n_update][version] = "1.1"
 
 projects[language_cookie][subdir] = "contrib"
+<<<<<<< HEAD
+projects[language_cookie][version] = "1.6"
+=======
 projects[language_cookie][version] = "1.9"
+>>>>>>> develop
```
Comment (senseca) [inline resources/multisite_drupal_standard.make]: Can you merge develop into branch in order to resolve confilcts?

**Comment by senseca** [comment]
* leaflet_widget module is installed in modules/ instead of modules/contrib. 

* The remove button is not working fine. Indeed, it removes the popup on edit form, but the popup (empty) is still there on view mode. Moreover, if there is already saved values for name and description, these values are not deleted.

**Comment by senseca** [comment]
* leaflet_widget module is installed in modules/ instead of modules/contrib. 

* The remove button is not working fine. Indeed, it removes the popup on edit form, but the popup (empty) is still there on view mode. Moreover, if there is already saved values for name and description, these values are not deleted.

File: resources/multisite_drupal_standard.make [hunk 7]
```diff
@@ -725,6 +725,13 @@ libraries[mpdf][download][url] = "http://mpdf1.com/repos/MPDF57.zip"
 libraries[mpdf][destination] = "libraries"
 
 
+<<<<<<< HEAD
```
Comment (senseca) [inline resources/multisite_drupal_standard.make]: Must be removed

File: resources/multisite_drupal_standard.make [hunk 8]
```diff
@@ -791,6 +798,7 @@ libraries[ec_resp_17_respond][directory_name] = "scripts"
 libraries[ec_resp_17_respond][download][type] = "get"
 libraries[ec_resp_17_respond][download][url] = "https://raw.githubusercontent.com/scottjehl/Respond/master/dest/respond.min.js"
 
+>>>>>>> develop
```
Comment (senseca) [inline resources/multisite_drupal_standard.make]: Must be removed

**Pull request merged**